lib/board.rb

```diff
@@ -0,0 +1,59 @@
+class Board
+  attr_accessor :cells
+  attr_reader :corners
+  @cells = []
+  def initialize
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+  end
+  def reset!
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+  end
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+    puts "*******************************************"
+  end
+  def position (cell_number)
+    @cells[cell_number.to_i - 1]
+  end
+  def full?
+   @cells.all? do |cell_number|
+     cell_number == "X" || cell_number == "O"
+    end
+  end
+  def turn_count
```

Comment:
@Juanriv45 for lines 27-35 we could simplify this a lot with the Ruby iterator `count`

example 

```ruby
@cells.count { |token| token == "X" || token == "O" }
```